packages-exp/auth-exp/src/core/credentials/phone.ts

```diff
@@ -121,7 +121,7 @@ export class PhoneAuthCredential extends AuthCredential {
     return obj;
   }
 
-  /** {@inheritdoc AuthCredential.fromJSON} */
+  /** Generates a Phone credential based on a plain object or a JSON string. */
```

Comment:
Literal?  Backtick if so.

Reply:
No literal here

Reply:
Should we lowercase "phone" in that case?